Simulator/Simulator/Simulator/python/plotting/network_activity/plot_fourieri_spike_activity.py

```diff
@@ -0,0 +1,120 @@
+import os
+import numpy as np
+from tqdm import tqdm
+
+from plot_network_spike_activity import get_args
+from plot_network_spike_activity import get_spikes_binaries_for_layer
+from plot_network_spike_activity import plot_data_and_lines
+from utils.utils_config import parse_yaml_config
+from utils.utils_simulator import parse_array_file
+from utils.utils_matplotlib import line_plot
+
+import matplotlib.pyplot as plt
+
+
+'''
+Almost the same code as plot_network_spike_activity.py but it will create a histogram-like of spike for each layer
+'''
+
+
+def get_fourier(signal):
+
```

Comment:
add link to youtube video from which you understood Fourier in general and in Python

Reply:
Added olso description